docker/nginx.conf

```diff
@@ -0,0 +1,11 @@
+server {
+  listen ${PORT};
```

Comment:
I think customization of port that nginx listen to is redundant - it should just use 8080 (or some other port).

When starting docker container you can always bind this port to some other port on the host machine.